jenkins/integ-test-notification.jenkinsfile

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+lib = library(identifier: 'jenkins@7.0.0', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/opensearch-project/opensearch-build-libraries.git',
+]))
+
+
+pipeline {
+    agent { label 'Jenkins-Agent-Ubuntu2004-X64-M58xlarge-Single-Host' }
+    options {
+        timeout(time: 1, unit: 'HOURS')
+        buildDiscarder(logRotator(daysToKeepStr: '90'))
+    }
+    triggers {
+        parameterizedCron('''
+            H */6 * * * %INPUT_MANIFEST=2.17.2/opensearch-2.17.2.yml
+            H */6 * * * %INPUT_MANIFEST=2.18.0/opensearch-2.18.0.yml
+            H */6 * * * %INPUT_MANIFEST=3.0.0/opensearch-3.0.0.yml
```

Comment:
Shall we update the input manifest generation workflow to update this section alongside check-for-build as well?

Reply:
Yes! Will take that up in next iteration.

Reply:
Could you create an issue for that since you seems to close #5038 in this PR.

Reply:
So you are going to run directly on agent node without docker container?

Reply:
Done https://github.com/opensearch-project/opensearch-build/issues/5083
Thanks!